.changeset/young-beans-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-lighthouse': minor
```

Comment:
Think this should be a `patch` unless it's a breaking change

Reply:
done